build/local.mk

```diff
@@ -113,11 +113,12 @@ test: $(THRIFTRW) __eval_packages ## run all tests
 .PHONY: cover
 cover: $(THRIFTRW) $(GOCOVMERGE) $(COVER) __eval_packages ## run all tests and output code coverage
 	PATH=$(BIN):$$PATH ./scripts/cover.sh $(PACKAGES)
-	go tool cover -html=cover.out -o cover.html
+	go tool cover -html=coverage.txt -o cover.html
 
-.PHONY: goveralls
-goveralls: cover $(GOVERALLS) ## run code coverage and upload to coveralls
-	PATH=$(BIN):$$PATH goveralls -coverprofile=cover.out -service=travis-ci
+.PHONY: codecov
+codecov: SHELL := /bin/bash
+codecov: cover ## run code coverage and upload to codecov.io
+	include_cov=coverage.txt bash <(curl -s https://codecov.io/bash)
```

Comment:
do you have a Q beyond what the comment describes?

I find the IoC here interesting :)